parallel-install/pkg/metadata/metadata.go

```diff
@@ -14,6 +16,9 @@ var metadataName = "kyma"
 //TODO it's in default ns cause at starting installation kyma-installer ns doesn't exist, this needs to be changed and metadata should be written to protected ns
 var metadataNamespace = "default"
 
+var initialInterval = time.Duration(3) * time.Second
+var maxElapsedTime = time.Duration(30) * time.Second
```

Comment:
Have you tested this values? How many retries do we get with this configuration? Maybe retries shouldn't be exponential in this case.

Reply:
adjusted values